TestI2sApplication/Program.cs

```diff
@@ -0,0 +1,24 @@
+using nanoFramework.Hardware.Esp32;
```

Comment:
please add as well the standard headers to the new files

Reply:
Copyright .NET Foundation and Contributors, etc? Will do